examples/compose-obect.js

```diff
@@ -0,0 +1,72 @@
+/*
+ * MinIO Javascript Library for Amazon S3 Compatible Cloud Storage, (C) 2021 MinIO, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// Note: YOUR-ACCESSKEYID, YOUR-SECRETACCESSKEY, my-bucketname and my-objectname
+// are dummy values, please replace them with original values.
+
+var Minio = require("../../dist/main/minio")
```

Comment:
Did you run the example script successfully?
First of all, I think there is a problem with the path. 
By looking at the path, it looks like you've assumed user will run the test from 2 level down from somewhere. The only location that makes sense to me is the `examples` directory. If that was the assumption, then the location is supposed to be:
```suggestion
var Minio = require("../dist/main/minio")
```
If I run from `examples` directory (or I also tried from the top level), I keep getting s3 error, code 400, which is interpreted as `Unknown Error` on our side, but the`400`, that is bad request. 
Please clarify.

Reply:
Since the testing of the API will involve more steps, i have added a file to demonstrate the same.

[composeObject full Example ](https://github.com/minio/minio-js/blob/f6b3be2b3330314fb3ece5fb85043bcd5c39dd52/examples/compose-object-test-example.js)